src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -16,6 +16,127 @@ limitations under the License.
 -->
 <configurator>
   <domain name="HVAC"/>
+
+  <bitmap name="ThermostatOccupancy" type="BITMAP8">
+    <cluster code="0x0201"/>
```

Comment:
Please do not expose new named entities that are not specced with those names.  Once they are in the spec, we can associate them with this cluster (and give them the spec names); until then we should not be exposing them.